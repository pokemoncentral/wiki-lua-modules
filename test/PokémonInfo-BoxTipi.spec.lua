-- "Test cases" for PokémonInfo-BoxAbil

local b = require("PokémonInfo-BoxTipi").boxTipi

-- Standard Pokémon
print(b({ args = { "Staraptor" } }))

-- Multigen
print(b({ args = { "Magneton" } }))

-- Alternative forms
print(b({ args = { "Rotom" } }))

-- Multigen and grouped alternative forms
print(b({ args = { "Gardevoir" } }))

-- Special children
print(b({ args = { "Ogerpon" } }))
