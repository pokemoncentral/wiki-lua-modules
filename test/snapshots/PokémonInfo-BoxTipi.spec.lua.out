<div class="roundy white-bg flex flex-row flex-wrap flex-main-stretch flex-items-center" style="padding: 0.2em;"><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #9FA19F;">[[Normale|<span style="color: #FFF;">Normale</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #81B9EF;">[[Volante|<span style="color: #FFF;">Volante</span>]]</span></div></div>
<div class="roundy white-bg flex flex-row flex-wrap flex-main-stretch flex-items-center" style="padding: 0.2em;"><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #60A1B8;">[[Acciaio|<span style="color: #FFF;">Acciaio</span>]]</span></div></div>
<div class="roundy white-bg flex flex-row flex-wrap flex-main-stretch flex-items-center" style="padding: 0.2em;"><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #704170;">[[Spettro|<span style="color: #FFF;">Spettro</span>]]</span><div class="small-text">Forma Rotom</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #E62829;">[[Fuoco|<span style="color: #FFF;">Fuoco</span>]]</span><div class="small-text">Rotom Calore</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #2980EF;">[[Acqua|<span style="color: #FFF;">Acqua</span>]]</span><div class="small-text">Rotom Lavaggio</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FD8FF;">[[Ghiaccio|<span style="color: #FFF;">Ghiaccio</span>]]</span><div class="small-text">Rotom Gelo</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #81B9EF;">[[Volante|<span style="color: #FFF;">Volante</span>]]</span><div class="small-text">Rotom Vortice</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #FAC000;">[[Elettro|<span style="color: #FFF;">Elettro</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FA129;">[[Erba|<span style="color: #FFF;">Erba</span>]]</span><div class="small-text">Rotom Taglio</div></div></div>
<div class="roundy white-bg flex flex-row flex-wrap flex-main-stretch flex-items-center" style="padding: 0.2em;"><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #EF4179;">[[Psico|<span style="color: #FFF;">Psico</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #EF70EF;">[[Folletto|<span style="color: #FFF;">Folletto</span>]]</span><div class="small-text">Tutte le forme</div></div></div>
<div class="roundy white-bg flex flex-row flex-wrap flex-main-stretch flex-items-center" style="padding: 0.2em;"><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FA129;">[[Erba|<span style="color: #FFF;">Erba</span>]]</span><div class="small-text">Maschera Turchese</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FA129;">[[Erba|<span style="color: #FFF;">Erba</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #2980EF;">[[Acqua|<span style="color: #FFF;">Acqua</span>]]</span><div class="small-text">Maschera Pozzo</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FA129;">[[Erba|<span style="color: #FFF;">Erba</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #E62829;">[[Fuoco|<span style="color: #FFF;">Fuoco</span>]]</span><div class="small-text">Maschera Focolare</div></div><div class="width-xl-50 text-center" style="box-sizing: border-box; padding: 0.2em;" ><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #3FA129;">[[Erba|<span style="color: #FFF;">Erba</span>]]</span><span style="padding: 0.1ex 0.3em 0.2ex 0.3em; background: #AFA981;">[[Roccia|<span style="color: #FFF;">Roccia</span>]]</span><div class="small-text">Maschera Fondamenta</div></div></div>
